native/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeGenerator.java

```diff
@@ -0,0 +1,203 @@
+            moduleMembers = populateRemaining(moduleMembers, existingNodes);
+            moduleMembers = addNewLine(moduleMembers, 1);
+
+        }
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        return SyntaxTree.from(textDocument);
+    }
+
+    private static boolean isDatabaseConfigurationEntry(KeyNode key) {
+        switch (key.toString().trim()) {
+            case KEY_USER:
+            case KEY_DATABASE:
+            case KEY_PASSWORD:
+            case KEY_HOST:
+            case KEY_PORT:
+                return true;
+            default:
+                return false;
+        }
+    }
+    private static int indexOf(String[] arr, String key) {
+        int index = 0;
+        for (String member : arr) {
+            if (key.equals(member)) {
+                return index;
+            }
+            index += 1;
+        }
+        return -1;
+    }
+
+    private static NodeList populateNodeList(NodeList moduleMembers) {
+        moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(KEY_HOST, DEFAULT_HOST, null));
+        moduleMembers = moduleMembers.add(SampleNodeGenerator.createNumericKV(KEY_PORT, DEFAULT_PORT, null));
+        moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(KEY_USER, DEFAULT_USER, null));
+        moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(KEY_PASSWORD, DEFAULT_PASSWORD, null));
+        moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(KEY_DATABASE, DEFAULT_DATABASE, null));
+        moduleMembers = moduleMembers.add(AbstractNodeFactory.createIdentifierToken(System.lineSeparator()));
+        return moduleMembers;
+    }
+
+    private static NodeList addNewLine(NodeList moduleMembers, int n) {
+        for (int i = 0; i < n; i++) {
+            moduleMembers = moduleMembers.add(AbstractNodeFactory.createIdentifierToken(System.lineSeparator()));
+        }
+        return moduleMembers;
+    }
+
+    private static NodeList populateRemaining(NodeList moduleMembers, ArrayList existingNodes) {
+        for (String key : nodeMap) {
+            if (!existingNodes.contains(key)) {
+                if (key.equals(KEY_PORT)) {
+                    moduleMembers = moduleMembers.add(SampleNodeGenerator.createNumericKV(key,
+                            defaultValues[indexOf(nodeMap, key)], null));
+                    existingNodes.add(key);
+                } else {
+                    moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(key,
+                            defaultValues[indexOf(nodeMap, key)], null));
+                    existingNodes.add(key);
+                }
+            }
+        }
+        return moduleMembers;
+    }
```

Comment:
```suggestion
    private static NodeList<DocumentMemberDeclarationNode> populateRemaining(
            NodeList<DocumentMemberDeclarationNode> moduleMembers, ArrayList<String> existingNodes) {
        for (String key : nodeMap) {
            if (!existingNodes.contains(key)) {
                if (key.equals(KEY_PORT)) {
                    moduleMembers = moduleMembers.add(createNumericKV(key,
                            defaultValues[indexOf(key)], null));
                } else {
                    moduleMembers = moduleMembers.add(createStringKV(key,
                            defaultValues[indexOf(key)], null));
                }
                existingNodes.add(key);
            }
        }
        return moduleMembers;
    }
```

Reply:
Updated in [53b94a5](https://github.com/ballerina-platform/persist-tools/pull/9/commits/53b94a557e6bdb6ac1800b8eb683bda757695cc5)